Assets/HoloToolkit/Input/Scripts/GazeStabilizer.cs

```diff
@@ -68,10 +68,10 @@ public struct GazeSample
 
         /// <summary>
         /// Updates the StableHeadPosition and StableHeadRotation based on GazeSample values.
-        /// Call this method with RaycastHit parameters to get stable values.
+        /// Call this method with Raycasthit parameters to get stable values.
         /// </summary>
-        /// <param name="position">Position value from a RaycastHit point.</param>
-        /// <param name="rotation">Rotation value from a RaycastHit rotation.</param>
+        /// <param name="position">Position value from a Raycasthit point.</param>
+        /// <param name="rotation">Roration value from a Raycasthit rotation.</param>
```

Comment:
Looks like you are reverting fixes made by Kurtis. Please update your fork and issue a new commit.